### DISCUSSION TITLE
How to access global system clipboard?

### DISCUSSION THREAD

**Discussion by GlassGruber** [Slackadays/Clipboard#90] [Q&A] [answered]
Sorry I've tried to check the docs but I wasn't able to find something about accessing the global system clipboard.

I'm on Windows, suppose I highlight some text in my browser, copy it either via `ctrl+c` or `right-click > copy`. How can I access that text with clipboard.exe? 

Thank you for this app, so simple yet effective! 
Hats off 🎩 👌

**Comment by Slackadays** [comment]
You can use `clipboard.exe` in anything in the terminal. So, you could pipe its contents to a different program, or show the contents of the global clipboard with `clipboard show`, or paste the contents to a file. Basically, there's no way to not access the global clipboard, as that's basked in to the Windows version of Clipboard. I hope this helps! :)

**Comment by GlassGruber** [comment, marked as answer]
My bad! I was just about to ask what is "clipboard.txt" when I realized that my output was quite different from the gif in the readme file, and ofc it is was since I was using version 0.20!

I've upgraded and now the command `show ` is far more descriptive, now what's in the buffer is displayed instead of clipboard.txt and everything is clear, very sorry and thank you for your answer and time!
